src/overlays/actors/ovl_En_Ma1/z_en_ma1.c

```diff
@@ -437,14 +433,25 @@ s32 EnMa1_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* p
 
 void EnMa1_PostLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3s* rot, void* thisx) {
     EnMa1* this = (EnMa1*)thisx;
-    Vec3f vec = D_80AA16B8;
+    Vec3f vec = { 800.0f, 0.0f, 0.0f };
 
     if (limbIndex == 15) {
         Matrix_MultVec3f(&vec, &this->actor.focus.pos);
     }
 }
 
 void EnMa1_Draw(Actor* thisx, PlayState* play) {
+    static void* sMouthTextures[] = {
+        gMalonChildNeutralMouthTex,
+        gMalonChildSmilingMouthTex,
+        gMalonChildTalkingMouthTex,
+    };
+
+    static void* sEyeTextures[] = {
+        gMalonChildEyeOpenTex,
```

Comment:
tiny thing, but you can remove the line break between these two. 
We always keep the stack of variables at the top of a function bunched together, and then put a line break to signal the code is now starting